binary_search_tree/binary_search_tree.py

```diff
@@ -0,0 +1,224 @@
+import sys
+
+
+class Node(object):
+    """
+    All data used in BST stored as object of Node class
+    data : It is the value store in object
+    left : it is left child of the object
+    right : It is right child of the object
+    """
+
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+
+class bst(object):
+
+    def __init__(self):
+        # here root is root of our BST tree which is initially None
+        self.root = None
+
+    def insert_node(self, data):
```

Comment:
Python codes are supposed to short. That is what makes python different from other languages.